packages/xcoobee-cookie-kit-react/src/CookieKitPopup.js

```diff
@@ -267,7 +317,10 @@ export default class CookieKitPopup extends React.PureComponent {
                     className={`${BLOCK}__checkbox`}
                   />
                 </div>
-                <div className={`${BLOCK}__cookie-title`}>
+                <div
+                  title="More info"
```

Comment:
This should be internationalized and localized.

Can we move the `title` prop after the `className` prop to keep props alphabetized to maintain project consistency?